test/e2e/e2e.test.js

```
@@ -73,15 +73,18 @@ for (let i = 0; i < testData.length; i++) {
-        const dataFetcherPromise = getDataFetcher(config, state.rpc, liquidityProviders);
+        const dataFetcherPromise = getDataFetcher(config, state.rpc, liquidityProviders, {
+            retryCountNext: Math.max(publicRpcs[chainId] * 2, 50),
+            timeout: 600_000,
+        });
 
         // run tests on each rp version
         for (let j = 0; j < rpVersions.length; j++) {
             const rpVersion = rpVersions[j];
 
-            it(`should clear orders successfully using route processor v${rpVersion}`, async function () {
+            it.only(`should clear orders successfully using route processor v${rpVersion}`, async function () {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)